### PR TITLE
feat: app details resource tree UI

### PR DESCRIPTION
**Pull request by RohitRaj011**
# Description

Update the UI for App Details Resource Tree as per latest design

[Figma](https://www.figma.com/design/9geLS2G5YwnOhejHhqmQor/DA--%3E-App-Details?node-id=8496-18164&node-type=frame&t=q9gnoZ3ukNXqUk8R-0)

Fixes https://github.com/devtron-labs/sprint-tasks/issues/891

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -1441,6 +1441,28 @@ button.anchor {
     }
 }
 
+.gui-yaml-switch.status-filter-button {
```
Comment (eshankvaish) [inline src/css/base.scss:1444]: let's move to specific file

File: src/components/v2/appDetails/k8Resource/k8resources.scss [hunk 2]
```diff
@@ -15,14 +15,13 @@
...
     min-height: 532px;
     overflow-y: auto;
 }
 
 .resource-node-wrapper {
-    min-height: calc(100vh - 168px);
+    min-height: calc(100vh - 120px);
 }
 
 .node-container-fluid {
```
Comment (eshankvaish) [inline src/components/v2/appDetails/k8Resource/k8resources.scss:27]: suggestion: let's clean this as well if possible

File: src/components/v2/appDetails/k8Resource/nodeType/nodeType.util.ts [hunk 3]
```diff
@@ -38,3 +39,18 @@ export const getNodeStatus = (node: iNode) => {
     }
     return ''
 }
+
+export const getFilteredPodStatus = (podStatusObj: NodePodStatus) => {
+    const podStatusKeys = Object.keys(podStatusObj)
+
+    if (podStatusKeys.length > 2 && podStatusObj['running'] === 0) {
```
Comment (eshankvaish) [inline src/components/v2/appDetails/k8Resource/nodeType/nodeType.util.ts:46]: let's add a comment what does this 2 signify

**Review by eshankvaish — CHANGES_REQUESTED** [review]

File: src/components/v2/appDetails/k8Resource/nodeType/nodeType.util.ts (hunk 3 above)
Comment (RohitRaj011) [reply src/components/v2/appDetails/k8Resource/nodeType/nodeType.util.ts:46]: existing code 
no idea about it

File: src/components/v2/appDetails/k8Resource/k8resources.scss (hunk 2 above)
Comment (RohitRaj011) [reply src/components/v2/appDetails/k8Resource/k8resources.scss:27]: this class is being used at multiple places

**Review by eshankvaish — DISMISSED** [review]

**Review by vivek-devtron — DISMISSED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**